.github/workflows/sdk-assets-component-component.yml

```diff
@@ -51,3 +51,23 @@ jobs:
       with:
         name: component
         path: sdk/assets/component
+
+    - name: Send IcM on failure
+      if: ${{ failure() && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/release') }}
```

Comment:
The release branch name will keep changing. You may need to add a contains 'release' filter